pkg/reloader/reloader.go

```diff
@@ -133,13 +151,15 @@ type Options struct {
 	// successful.
 	RuntimeInfoURL *url.URL
 
-	// CfgFile is a path to the prometheus config file to watch.
+	// CfgFile is a path to the Prometheus config file to watch.
 	CfgFile string
 	// CfgOutputFile is a path for the output config file.
 	// If cfgOutputFile is not empty the config file will be decompressed if needed, environment variables
 	// will be substituted and the output written into the given path. Prometheus should then use
 	// cfgOutputFile as its config file path.
 	CfgOutputFile string
+	// SubCfgs is a collection of path to Prometheus sub-configs to watch.
+	SubCfgs []SubConfigOptions
```

Comment:
Thanks! I think I know why you chosen "subcfgs" but I think it could be more clear if we just called "CfgDirs". Then we can explain that "CfgDirs" allows storing multiple files that will be processed in the same way as "CfgFile" (so watched, compression handling and env substitution).

Reply:
Alternatively `Cfgs` if we want more flexibility on what's in `CfgOption`

Reply:
Essentially we want to also use it for recording rules, not only sub Prometheus config files, so "config" as any config resource (Prometheus config, sub scrape config, rule file, etc) fits well.

Reply:
I kept it `CfgDirs` because the object is named `CfgDirOption`. I feel it's more clear this way.